File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/gpt-neo-2.7/find_first_in_sorted.java
--- 
+++ 
@@ -2,7 +2,7 @@
     int lo = 0;
     int hi = arr.length;
 
-    while (lo <= hi) {
+    while (lo < hi) {
         int mid = (lo + hi) / 2; // check if this is floor division
 
         if (x == arr[mid] && (mid == 0 || x != arr[mid-1])) {

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/gpt-neo-125/find_first_in_sorted.py
--- 
+++ 
@@ -2,7 +2,7 @@
     lo = 0
     hi = len(arr)
 
-    while lo <= hi:
+    while lo < hi:
         mid = (lo + hi) // 2
 
         if x == arr[mid] and (mid == 0 or x != arr[mid - 1]):
